bundle-core/src/main/proto/BundleService.proto

```diff
@@ -11,8 +11,7 @@ import "FileService.proto";
 
 message BundleMetaData {
   string bid = 1;
-  string senderId = 2;
-  string sender = 3;
+  BundleSender sender = 4;
```

Comment:
why do we need this to be numbered as parameter 4 and not 2?
is it for backward compatibility for older data?

Reply:
yeah, just to not overlap. at least this way it will fail with a null pointer rather than some crazy decoding :)